addons/payment/views/portal_templates.xml

```diff
@@ -16,6 +16,21 @@
         </xpath>
     </template>
 
+    <template id="portal_my_home_payment" name="Show Payment Method" customize_show="True" inherit_id="portal.portal_my_home" priority="60">
```

Comment:
It's cool stuff to add a card for payment methods :)

I believe you can now remove the `payment.pay_meth_link` template since it does the same job but looks worse.